packages/pd-manager/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@sphereon/ssi-sdk.pd-manager",
+  "version": "0.23.4",
+  "source": "src/index.ts",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "veramo": {
+    "pluginInterfaces": {
+      "IPDManager": "./src/types/IPDManager.ts"
+    }
+  },
+  "scripts": {
+    "build": "tsc",
+    "build:clean": "tsc --build --clean && tsc --build"
+  },
+  "dependencies": {
+    "@sphereon/ssi-sdk.data-store": "workspace:*",
+    "@sphereon/pex": "^3.3.3",
```

Comment:
checking that we cannot move this and "@sphereon/pex-models" to devDependencies

Reply:
Can we? IPresentationDefinition is exposed in the exported model

Reply:
No it's exported